chain/default_syncer.go

```diff
@@ -110,7 +110,7 @@ func (syncer *DefaultSyncer) getBlksMaybeFromNet(ctx context.Context, blkCids []
 // that interacts with the network. It does NOT add tipsets to the chainStore..
 func (syncer *DefaultSyncer) collectChain(ctx context.Context, tipsetCids types.SortedCidSet) ([]types.TipSet, error) {
 	var chain []types.TipSet
-	defer logSyncer.Info("chain synced")
+	defer logSyncer.Info("chain fetch from network complete")
 	for {
 		var blks []*types.Block
 		// check the cache for bad tipsets before doing anything
```

Comment:
Blocking: down at line 144 we should change the log from "syncing" to "fetching"

Reply:
Also down at line 144 we should count up in terms of blocks fetched, not down in terms of block height.

Reply:
nice catch, will do.